Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -415,20 +421,22 @@ logOutPopUpModelConfig state =
               , background = state.data.config.primaryBackground
               , text = (getString LOGOUT_)
               }
+            , popUpStatus = spy "logOutPopUpModelConfig state::" state.data.popUpConfig.status
+            , actionType = state.data.popUpConfig.actionType
             }
       in
         popUpConfig'
     ST.TipsPopUp -> PopUpModal.config{
           optionButtonOrientation = "VERTICAL"
           , dismissIconMargin = Margin 0 0 14 13
           , dismissIconVisibility = if isLocalStageOn ST.QuoteList then GONE else VISIBLE
-          , backgroundClickable = true
+          , backgroundClickable = false
```

Comment:
why is this changed?

Reply:
because when background clickable is true and dismissPopUp is also true at that time `DismissPopUp` is the action which gets triggered and in controller we are just doing continue state as the localStage is `QuoteList`

Following is the. code snippet for the same.
`eval (PopUpModalAction (PopUpModal.DismissPopup)) state = do
  let newState = if (isLocalStageOn QuoteList) then state else state{props{isPopUp = NoPopUp, customerTip{tipActiveIndex = 1,tipForDriver = 10, isTipSelected = false} }}
  continue newState`